File: src/java/eu/sig/training/ch03/withmapandtypes/DutchFlag.java
package eu.sig.training.ch03.withmapandtypes;

import java.awt.Color;
import java.util.Arrays;
import java.util.List;

// tag::DutchFlag[]
public class DutchFlag implements Flag {
    public List<Color> getColors() {
        return Arrays.asList(Color.RED, Color.WHITE, Color.BLUE);
    }
}
// end::DutchFlag[]

File: src/csharp/eu/sig/training/ch03/withmapandtypes/DutchFlag.cs
using System.Collections.Generic;
using System.Drawing;

namespace eu.sig.training.ch03.withmapandtypes
{

    // tag::DutchFlag[]
    public class DutchFlag : IFlag
    {
        public IList<Color> Colors
        {
            get
            {
                return new List<Color> { Color.Red, Color.White, Color.Blue };
            }
        }
    }
    // end::DutchFlag[]
}
